### DISCUSSION TITLE
DNS records IP ddresses

### DISCUSSION THREAD

**Discussion by vamrishiaj** [learndevopsonline/learndevopsonline#1460] [Q&A]
Hello,
    I have bought a domain name, attached it to AWS Hosted zone. Updated all 4 addresses in hostinger as well. My issue is I am creating DNS records. which IP address to give for all instances. Public IP or Private IP? (or)only frontend public and rest private? someone please help

**Comment by sreedharm07** [comment]
Hi @vamrishiaj  
give private ip for all  components
only to ssh we use public ip address

**Reply by vamrishiaj** [reply]
thank you bro
